### PR TITLE
issue #7

### PR DESCRIPTION
**Pull request by otacon94**
Please review that :D

# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/Sid200026/ToxicBot/blob/master/CONTRIBUTING.md)?

- [X] Yes
- [ ] No

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update
- [ ] Documentation Update

### CODE REVIEW

File: ToxicBot/app.py [hunk 1]
```diff
@@ -17,8 +17,10 @@
 DISCORD_BOT_TOKEN = config.get('Discord', 'BOT_TOKEN')
 COMMAND_PREFIX = "/"
 
+print(DISCORD_BOT_TOKEN)
```
Comment (Sid200026) [inline ToxicBot/app.py:20]: Please remove this print statement

File: ToxicBot/commands/commands.py [hunk 2]
```diff
@@ -13,3 +14,20 @@ async def info(self, ctx):
...
+        await ctx.send(HELP_MESSAGE.format(username=member.name))
+
+    @commands.command()
+    @commands.dm_only()
+    async def report(self, ctx):
+        member = ctx.author
+        channel = ctx.channel
+        embed = discord.Embed(title='GitHub - ToxicBot',
+                       url='https://github.com/Sid200026/ToxicBot/issues',
+                       description='Report issues and bugds')
```
Comment (Sid200026) [inline ToxicBot/commands/commands.py:32]: bugds -> bugs

File: ToxicBot/constants/messages.py [hunk 3]
```diff
@@ -14,6 +14,18 @@
 Any message that I deem toxic, insulting, threatening etc will be removed and the author will be given a warning. \
 """
 
+HELP_MESSAGE = """
+Hi {username}, this is Mr Toxic Bot. \n
+Here's a list of commands you can use to chat with me: \n
+- /report - With this command you'll receive the link of the GitHub page of the project to report bug and issues \n
```
Comment (Sid200026) [inline ToxicBot/constants/messages.py:20]: bug -> bugs

File: ToxicBot/constants/messages.py [hunk 4]
```diff
@@ -14,6 +14,18 @@
...
+- /info - You'll receive some info about this me :D \n
```
Comment (Sid200026) [inline ToxicBot/constants/messages.py:21]: remove this

File: ToxicBot/constants/messages.py [hunk 5]
```diff
@@ -14,6 +14,18 @@
...
+- /help - You'll receive the list of the commands
```
Comment (Sid200026) [inline ToxicBot/constants/messages.py:22]: the list of the commands -> the list of commands

**Review by Sid200026 — COMMENTED** [review]
I have requested some minor changes in the commits. Please make the changes and I'll merge this branch into master.

File: ToxicBot/app.py (hunk 1 above)
Comment (otacon94) [reply ToxicBot/app.py]: sorry, I did it for debugging purposes and I forget to remove :s

**Comment by otacon94** [comment]
> I have requested some minor changes in the commits. Please make the changes and I'll merge this branch into master.

Done :D

**Comment by Sid200026** [comment]
Thank you for contributing. All the best

**Pull request merged**